### PR TITLE
release 3.0.0 - Implement HTTP Proxy

### PR DESCRIPTION
**Pull request by linuradu**
**Implemented**

- Implemented support for HTTP Proxy

### CODE REVIEW

**Comment by linuradu** [comment]
@2pd @aisling-2 Can you please help with a code review?

Best regards,
Radu Linu

**Comment by linuradu** [comment]
@bnbot @aisling-2 @2pd @jonte-z Can you please give me an update in regards to this pull request?

Best regards,
Radu

**Comment by aisling-2** [comment]
Hi @linuradu, it's in our queue to review, we'll update this PR when we get into it. Thank you for your contribution!

**Comment by jontstaz** [comment]
.gz " ] // < kind >finished</ kind > } } }

最后做了一遍的检查，二次编译可以。

今天是某日。

File: pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
     <groupId>io.github.binance</groupId>
     <artifactId>binance-futures-connector-java</artifactId>
-    <version>2.0.0</version>
+    <version>2.2.0</version>
```
Comment (jonte-z) [inline pom.xml:9]: Should be Version 2.1.0

Comment (jonte-z) [reply pom.xml]: I made the change and committed. I will be finished with the review very shortly.

File: README.md [hunk 2]
```diff
@@ -119,6 +119,52 @@ output:
...
+```
+
+To undo `Proxy`, use `unsetProxy()` before submitting requests to binance:
+
+```java
+client.unsetProxy();
+logger.info(client.market().time());
+```
+
+Complete examples are available at `test/examples/proxy` folder.
```
Comment (jonte-z) [inline README.md:166]: Missing the examples, please commit them. Also the correct path for examples should be `test/java/examples/cm_futures/` and/or `test/java/examples/um_futures`

Comment (linuradu) [reply README.md]: @jonte-z I updated the README.md and also added the HTTP Proxy examples. 
Please check the last commit.

Comment (jonte-z) [reply README.md]: Thanks, checking.

**Review by jonte-z — APPROVED** [review]
Thanks for the changes. Looks good to me now. Approved

**Comment by linuradu** [comment]
Many thanks @jonte-z for the review! 

Can you please tell me when will be merged `rc-3.0.0` into the `main` branch and deployed to `https://mvnrepository.com/`?

**Pull request merged**